src/renault_api/cli/charge/control.py

```diff
@@ -47,5 +47,31 @@ async def start(
     vehicle = await renault_vehicle.get_vehicle(
         websession=websession, ctx_data=ctx_data
     )
-    response = await vehicle.set_charge_start()
+    details = await vehicle.get_details()
+    if details.get_model_code() == "XBG1VE":
+        response = await vehicle.set_charge_pause_resume("resume")
+    else:
+        response = await vehicle.set_charge_start()
```

Comment:
I think it is better to keep the original method (simply set_charge_start) and move the logic there.
```suggestion
    response = await vehicle.set_charge_start()
```